testing/test_framework/src/firebase_test_framework.cc

```diff
@@ -309,13 +309,17 @@ std::vector<std::string> ArgcArgvToVector(int argc, char* argv[]) {
 
 char** VectorToArgcArgv(const std::vector<std::string>& args_vector,
                         int* argc) {
-  char** argv = new char*[args_vector.size()];
+  // Create `argv` one element larger than strictly required since gtest expects
```

Comment:
IIUC, this is a POSIX requirement -- if so, it should probably be mentioned in this comment (otherwise, it reads as if it's a requirement imposed by Googletest).